appveyor/mozillaSyms.py

```diff
@@ -0,0 +1,102 @@
+# Based on code from https://gist.github.com/luser/2ad32d290f224782fcfc
+
+"""Script to convert and upload appropriate NVDA debug symbols to Mozilla crash-stats.
+This should just be run as a script with no arguments.
+It expects the crash-stats auth token to be placed in the mozillaSymsAuthToken environment variable.
+To update the list of symbols uploaded to Mozilla, see the DLL_NAMES constant below.
+"""
+
+from __future__ import print_function
+import argparse
+import os
+import subprocess
+import sys
+import zipfile
+import requests
+
+SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
+DUMP_SYMS = os.path.join(SCRIPT_DIR, "dump_syms.exe")
+NVDA_SOURCE = os.path.join(os.path.dirname(SCRIPT_DIR), "source")
+NVDA_LIB = os.path.join(NVDA_SOURCE, "lib")
+NVDA_LIB64 = NVDA_LIB + "64"
+ZIP_FILE = os.path.join(SCRIPT_DIR, "mozillaSyms.zip")
+URL = 'https://crash-stats.mozilla.com/symbols/upload'
+
+# The dlls for which symbols are to be uploaded to Mozilla.
+# This only needs to include dlls injected into Mozilla products.
+DLL_NAMES = [
+	"IAccessible2Proxy.dll",
+	"minHook.dll",
+	"nvdaHelperRemote.dll",
+	"VBufBackend_adobeFlash.dll",
+	"VBufBackend_gecko_ia2.dll",
+]
+DLL_FILES = [f
+	for dll in DLL_NAMES
+	# We need both the 32 bit and 64 bit symbols.
+	for f in (os.path.join(NVDA_LIB, dll), os.path.join(NVDA_LIB64, dll))]
+
+class ProcError(Exception):
+	def __init__(self, returncode, stderr):
+		self.returncode = returncode
+		self.stderr = stderr
+
+def check_output(command):
+	proc = subprocess.Popen(command,
+		stdout=subprocess.PIPE,
+		stderr=subprocess.PIPE)
+	stdout, stderr = proc.communicate()
+	if proc.returncode != 0:
+		raise ProcError(proc.returncode, stderr)
+	return stdout
+
+def processFile(path):
+	try:
+		stdout = check_output([DUMP_SYMS, path])
+	except ProcError as e:
+		print('Error: running "%s %s": %s' % (DUMP_SYMS, path, e.stderr))
+		return None, None, None
+	bits = stdout.splitlines()[0].split(' ', 4)
+	if len(bits) != 5:
+		return None, None, None
+	_, platform, cpu_arch, debug_id, debug_file = bits
+	sym_file = debug_file[:-4] + '.sym'

```

Comment:
Perhaps we know this since the files are all listed above? In that case I would extract this `-4` value and name it something like `numberOfCharsInExtension` and put it next to the list of files with a comment that says that all the files have the same number of characters in the extension.